README.md

```diff
@@ -32,6 +32,14 @@ $ cp .env.template .env  # (first time only)
 
 The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like development mode (which also enables features like hot reloading when you make a file change). There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
 
+You will need to setup a Trello account, create a ToDo and a Complete board and change the following keys in the .env file:
+
+SECRET_KEY = Key generated on the Trello site
```

Comment:
Flask uses `SECRET_KEY` to sign session cookies (that's why you needed `SECRET_KEY` set for exercise 1), we're not using session storage any more so it works fine to use it for something else, but just to avoid confusion it might be better to name it something else, e.g. `TRELLO_KEY`